README.md

```diff
@@ -1,28 +0,0 @@
-# getGitHub
-> A similar to Github - For Developers - setAttribute
->Done by: Samer, Waleed, Mo'ath and Kefah ...
-
-### What is getGitHub?
-A simple look alike Github site, which shows information of the developers who are members in Github.
-### What's new from previous projects?
-The new approach of getting information from calling server, connections between frontend and backend of the site.
-### How does it work?
-The user writes the name of the developer in search input box, then click search to get the information of the wanted developer such as: Avatar, Name, Login Name, Repos, Stars and so on.
-What is the sites structure and files?
-THE SITE CONSISTS OF:
-#### 1. FRONTEND SIDE:
-public folder having index.html, test.js and contains:
-**css folder** : style.css.  
-**js folder** : index.js.
-#### 2. BACKEND SIDE:
-**src folder**  having ```apiReq.js```, ```handlers.js```, ```router.js```, ```server.js``` and ```test.js```
-### How to Install and run?
-1. Clone the repo from [Here](http://github.com/facg2/getGitHub) .
-2. Open the terminal and move to the project's folder >cd getGitHub.
-3. Run atom application to open the projects files and folders using terminal >atom .
-4. In atom open the index.html by right click on the file and choose Show in File Manager.
-5. Right click on index.html and open using Chrome.
-6. To run server.js install dependencies using ```>npm init``` .
-7. To run the program on local server ```>node server.js``` open the localhost:3500 in Chrome Browser to run the project.
-
```

Comment:
It's not necessary to be so detailed, or mention specific products like atom or chrome. Or to explain how to read files etc.

##### To setup:
- `npm install`

##### To run:
- `node server.js`
- Visit `localhost:3500`